### ISSUE TITLE
Error when running `guild run --stop-after N` 

### ISSUE THREAD

**Issue by lucas-ubm** [guildai/guildai#155]
After trying to start a run using the stop after parameter it crashes giving the following error log:

```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "./venv/lib/python3.7/site-packages/guild/op_util_legacy.py", line 161, in _err_tee_run
    self._gen_tee_run(self._proc.stderr, sys.stderr, 1)
  File "./venv/lib/python3.7/site-packages/guild/op_util_legacy.py", line 185, in _gen_tee_run
    os_write(output_fileno, buf)
OSError: [Errno 9] Bad file descriptor
```
And 
```
Traceback (most recent call last):
  File "./venv/lib/python3.7/site-packages/guild/op.py", line 238, in _op_watch_proc
    return _proc_wait(proc, stop_after)
  File "./venv/lib/python3.7/site-packages/guild/op.py", line 267, in _proc_wait
    return op_util.wait_for_proc(proc, stop_after)
AttributeError: module 'guild.op_util' has no attribute 'wait_for_proc'
```

**Comment by gar1t** [comment]
Thank you for the report - and apologies for the long delay in replying!

This is resolved in master and will be available as a fix in rc10, due out this week.

I created an issue resolution doc here:

https://github.com/guildai/issue-resolution/tree/master/155-error-when-running-guild-run-stop-after

We use these to verify the issue and track the resolution with actual runnable code.

**Comment by gar1t** [comment]
This should be fixed in 0.7.0. If you still see this behavior, please reopen this issue and we'll sort it out. Thanks again for the report!
